### PR TITLE
move fs2 to v1.0.0 tag

### LINKED ISSUE #804: move fs2 to 1.0.0
1.0.0 was released 14 days ago. we are presently on 1.0.0-M5

moving to 1.0.0 is generally desirable, but would also fix it on JDK 11 by pulling in https://github.com/functional-streams-for-scala/fs2/commit/226ff68e34bff56224ecef1c18d6bee4fe4cb0a2#diff-736c8aaa74103281c32f371842cf994bR41

### PR DESCRIPTION
**Pull request by SethTisue**
fixes #804

### CODE REVIEW

**Comment by SethTisue** [comment]
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3658/

**Comment by SethTisue** [comment]
(hopefully no downstream projects break, but if they do, we should probably suggest/insist they upgrade)

**Comment by SethTisue** [comment]
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3661/

**Comment by SethTisue** [comment]
hmm, maybe I should move cats-effect to 1.0.0, also

**Comment by SethTisue** [comment]
oh, we are on cats-effect 1.0.0 already. an out-of-date comment made me think we weren't

**Comment by SethTisue** [comment]
run 3661 just had `failed: sttp, http4s` so I've tried unfreezing/unforking those, new run will be:

https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3661/

**Comment by SethTisue** [comment]
~~https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3668~~
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3675/

**Comment by SethTisue** [comment]
squashed, now that this is clearly about to cross the finish line (just getting http4s green now)

https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3695/

File: configs/community.dbuild [hunk 1]
```diff
@@ -2302,17 +2286,20 @@ build += {
...
-    // I got some compile error here (September 2018) that seemed unlikely to be
-    // significant. it's fine, we don't need to have every last subproject
-    extra.exclude: ["monix", "okhttpBackendMonix", "asyncHttpClientBackendMonix"]
+    extra.exclude: [
+      // I got some compile error here (September 2018) that seemed unlikely to be
+      // significant. it's fine, we don't need to have every last subproject
+      "monix", "okhttpBackendMonix", "asyncHttpClientBackendMonix"
+      // doesn't appear to have caught up with fs2-reactive-streams no longer being in a separate repo
+      // (October 2018)
+      "asyncHttpClientBackendFs2"
```
Comment (kubukoz) [inline configs/community.dbuild:2301]: It has, since https://github.com/softwaremill/sttp/pull/151 and https://github.com/softwaremill/sttp/pull/153

Comment (SethTisue) [reply configs/community.dbuild:2301]: trying at https://github.com/scala/community-builds/pull/837 to re-add it

**Pull request merged**